### DISCUSSION TITLE
tictactoe.js - command

### DISCUSSION THREAD

**Discussion by rishanbenahalkar** [SudhanPlayz/Discord-MusicBot#974] [Show and tell]
Version: v13.8.0
Packages: discord.js, discord-tictactoe

**Preview:**
![image](https://user-images.githubusercontent.com/79592017/177565862-85bda279-694e-4b11-9db1-30bdec2ed764.png)
![image](https://user-images.githubusercontent.com/79592017/177565985-266c555f-db51-49c2-893b-ce1ede150399.png)
![image](https://user-images.githubusercontent.com/79592017/177566038-abfb73a5-e49b-4301-a362-dfb6a468960f.png)
![image](https://user-images.githubusercontent.com/79592017/177566244-39e1adb0-05f0-423e-ac83-8e5c019f58ba.png)

**Notes:** 
If an opponent is not provided you will be playing against the AI 
If an opponent is provided, it will ping the opponent as seen in image two 
If the opponent does not click either accept or deny to the game in the adequate time, it will show an embed as seen in image four 
If you do not provide an opponent and are now playing with the AI and you are inactive it will show an embed as seen in image three

```js
const SlashCommand = require("../../lib/SlashCommand");
const TicTacToe = require("discord-tictactoe");
const game = new TicTacToe({ language: 'en'}); 

const command = new SlashCommand()
  .setName("tictactoe")
  .setDescription("Play tictactoe")
  .addUserOption((option) =>
    option
      .setName("opponent")
      .setDescription("The opponent to fight against")
      .setRequired(false)
  )
  .setRun(async (client, interaction) => {
    game.handleInteraction(interaction);
  });

  module.exports = command;
```

**Comment by sprucecellodev125** [comment]
Why would we need tic tac toe command in music bot?

**Reply by rishanbenahalkar** [reply]
who knows, im just posting, maybe you can use this to waste some extra time you got, this is something that is very far off from what a music bot is but you could always add it to your own private copy of Discord-MusicBot.

**Comment by sprucecellodev125** [comment]
This is music bot, not multipurpose bot

**Reply by DevAmiyo** [reply]
It's not for adding into the repository. It is just there so that if anyone wants to add it to their bot they can.

Am I right @XstreamSpeed?

**Reply by rishanbenahalkar** [reply]
Exactly.
